src/Server/Remote/PhpSecLib.php

```diff
@@ -50,51 +50,41 @@ public function connect()
 
         switch ($serverConfig->getAuthenticationMethod()) {
             case Configuration::AUTH_BY_PASSWORD:
-
-                $result = $this->sftp->login($serverConfig->getUser(), $serverConfig->getPassword());
-
+                $authParams = [$serverConfig->getUser(), $serverConfig->getPassword()];
```

Comment:
@elfet Let's leave it for removing code duplication in v4.x. Scrutinizer likes it :)